packages/tailwind/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "peerDependencies": {
     "@astrojs/starlight": ">=0.7.2",
```

Comment:
This will need bumping manually to `>=0.9.0` as a major change in the Tailwind plugin. I think we need to do that in the Changesets release PR.